db/migrate/20181121182827_create_charges.rb

```diff
@@ -0,0 +1,11 @@
+class CreateCharges < ActiveRecord::Migration[5.1]
+  def change
+    create_table :charges do |t|
+      t.integer :user_id
```

Comment:
I think here you can use `t.references :user`: not only will it create the appropriate `user_id` field, but also the appropriate db index on `user_id`, that will allow for fast search by user_id on this table.

Reply:
Oh, I'll try that instead!